lib/typescript.d.ts

```diff
@@ -4487,9 +4487,9 @@ declare namespace ts {
          * @param compilationSettings Some compilation settings like target affects the
          * shape of a the resulting SourceFile. This allows the DocumentRegistry to store
          * multiple copies of the same file for different compilation settings.
-         * @parm scriptSnapshot Text of the file. Only used if the file was not found
```

Comment:
for future references, please do not edit files under `lib` manually. these are generated as part of the build.